tsconfig.base.esm.json

```diff
@@ -1,7 +1,8 @@
 {
+  "$schema": "https://json.schemastore.org/tsconfig",
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
     "module": "es2022",
-    "moduleResolution": "node16"
+    "moduleResolution": "bundler"
```

Comment:
I don't have a good sense of the potential impact of this. (https://www.typescriptlang.org/tsconfig/#moduleResolution)